.gitignore

```diff
@@ -78,3 +78,8 @@ target/
 
 # Vagrant
 .vagrant
+
+# IntelliJ project files
+.idea
+*.iml
+out
```

Comment:
Legit but not needed. I'll remove it.
IntelliJ -specific folder for [generated .class files and artifacts](http://stackoverflow.com/a/28231127)